src/main/java/net/ossindex/gradle/OssIndexPlugin.java

```diff
@@ -22,20 +23,41 @@
 
 public class OssIndexPlugin implements Plugin<Project> {
 
+    public static JunitXmlReportWriter junitXmlReportWriter = null;
+
+    public static String junitReport = null;
+    public static Integer instanceId;
+
+    private static AuditExtensions settings = null;

```

Comment:
I think I am getting really picky here, but since this is only used in one place you may as well just instantiate a local field down there instead. I'll leave a note at that location as well...

Reply:
Yes I originally had a fairly clunky and horrible bunch of code around this. I've changed this to a local var